src/lib.rs

```diff
@@ -1,7 +1,10 @@
 use std::borrow::Cow;
 
+pub mod ascii_fold;
+
 pub fn normalize_word(source: &str) -> Cow<str> {
-    let mut dest = source.to_lowercase();
+    let mut dest = ascii_fold::latinize(source);
+    dest.make_ascii_lowercase();
```

Comment:
nitpick: it might read a little nicer as 
```
let mut dest = ascii_fold::latinize(source).to_lowercase();
```

Reply:
I was trying to avoid the extra allocation. So when you call `latinize()`, it returns a new string. The inspiration one allowed for reuse of a string for output, but I hate side-effecting functions. But if you call `latinize()` it will allocate a string, then if you call `to_lowercase()` it will allocate another one. The other thing here is that `to_lowercase` handles UTF-8 but is less efficient. The `make_ascii_lowercase()` function is able to be much more efficient as well as working in place. But you can only use it if you know your input is just ASCII -- which it is now that it's been latinized/ascii folded.